scripts/preparare-deploy-source-code.sh

```diff
@@ -66,17 +67,19 @@ update_source() {
   # git commit fails if the commit is empty, which makes Travis build fail.
   git diff-index --quiet HEAD || \
       git commit -a \
-          -m "Pseudo-Merge $gitlog" \
+          -m "Pseudo-Merge $gittilte" \
```

Comment:
tilte -> title

Reply:
But the typo ruins everything. Thanks. I think I'm too lazy to properly test this change, though.